crates/masking/src/secret.rs

```diff
@@ -152,3 +152,9 @@ where
         SecretValue::default().into()
     }
 }
+
+impl ToString for Secret<String, crate::strategy::ApiKey> {
+    fn to_string(&self) -> String {
+        self.inner_secret.clone()
+    }
+}
```

Comment:
Would this prevent `Secret<String>` types from being logged/formatted in plaintext when passed to either the logging macros or the `format!()` macro?

Reply:
Yes, for the purpose of logging and formatting, the `Display` and `Debug` implementation are used accordingly. In this case we are performing the same action as `peek`. But, having `ToString` on a secret isn't optimal in every case, thus this implementation is only limited to `ApiKey` masking strategy

Reply:
Already we having `peek`. Why taking a different direction for `Apikey`?